processing/data_collection/gazette/spiders/ma_sao_luis.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+import dateparser
+from datetime import datetime
+from scrapy_splash import SplashRequest
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+from scrapy.shell import inspect_response
+
+
+class MaSaoLuisSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '2111300'
+    name = 'ma_sao_luis'
+    allowed_domains = ['www.semad.saoluis.ma.gov.br']
+    start_urls = ['http://www.semad.saoluis.ma.gov.br:8090/easysearch/']
+
+    lua_script = """
```

Comment:
I don't like this script mixed with the Spider code. Maybe removing it from this file and create a separated `.lua` file would be better? What do you think?